civihr_default_theme/scss/base/_variables.scss

```diff
@@ -163,3 +163,8 @@ $crm-scrollbar-thumb-color: $crm-gray-light-blue !default;
 $crm-customselect-height:        33 !default;
 $crm-customselect-caret-width:   32 !default;
 $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
+
+//== Forms
+//
+//##
+$input-border:              $crm-color-neutral !default;
```

Comment:
is `$input-border` a default bootstrap SASS variable? If yes, what impact does this have on the SSP theme as a whole?